e2e/resources/forms/custom-date-form.ts

```diff
@@ -93,7 +93,7 @@ export const customDateFormAPS1 = `{
                                     "existingColspan": 1,
                                     "maxColspan": 2
                                 },
-                                "dateDisplayFormat": "YY-M-D",
+                                "dateDisplayFormat": "yy-M-d",
```

Comment:
this format is coming from the backend, it should remain the same not to break all existing forms

Reply:
I remember making that format change in custom-date.form.ts because some e2es were breaking because of the format. I will revert it and test it again.

Reply:
I have reverted the changes from _'custom-date-form.ts'_. 
For the e2e to run properly, we still need the _dateDisplayFormat_ to be _'yy-M-d'_ so that the form config can be rendered as a form  (as the date widget now supports date-fns formats).

So, I have added an `updateDateFormat()` method in _'form-demo.page.ts'_ to change dateFormat while running the e2e test case.

Reply:
Upon re-analyzing, I think that this approach would be wrong too because ideally, we should not change the form config to make the e2es work because the functionality would still be impacted unless we change the 'dateDisplayFormat' from the backend as well. So for now, I will revert the fixes I did for this (`updateDateFormat()`).

Also, the 5 e2es that are failing in the build ( C268818, C268819, C290040, C313200, C310199 ) are fetching forms from backend ( e2e/resources/activiti7/candidatebaseapp.zip, e2e/resources/activiti7/simpleapp.zip, e2e/resources/apps/WidgetApps.zip ). We probably need to get the 'dateDisplayFormat' changed in these form jsons as well. 
@DenysVuika @eromano